webui/react/src/pages/TrialLogs.tsx

```diff
@@ -87,11 +86,23 @@ const TrialLogs: React.FC = () => {
     if (logsRef.current) logsRef.current?.addLogs(pollingLogsResponse.data);
   }, [ logIdRange, pollingLogsResponse.data ]);
 
+  const downloadLogs = useCallback(() => {
+    return downloadTrialLogs(id).catch(e => {
```

Comment:
**nit:**
```
  return downloadTrialLogs(id).catch(e => {
    handleError({
      ...
    });
  });
```

Reply:
sidenote, on serious problem the page probably doesn't get here and crashes.